x-pack/plugin/spatial/src/test/java/org/elasticsearch/xpack/spatial/search/aggregations/GeoLineAggregatorTests.java

```diff
@@ -942,12 +942,12 @@ private <A extends MultiBucketsAggregation, C extends Aggregator> void testCase(
             if (timeSeries) {
                 fieldTypes.add(TimeSeriesIdFieldMapper.FIELD_TYPE);
                 fieldTypes.add(new DateFieldMapper.DateFieldType("@timestamp"));
-                fieldTypes.add(new DateFieldMapper.DateFieldType("time_field"));
                 var metricType = randomBoolean() ? POSITION : null; // metric type does not affect geo_line behaviour
                 fieldTypes.add(new GeoPointFieldMapper.GeoPointFieldType("value_field", metricType, TIME_SERIES));
             } else {
                 fieldTypes.add(new GeoPointFieldMapper.GeoPointFieldType("value_field"));
             }
+            fieldTypes.add(new DateFieldMapper.DateFieldType("time_field"));
```

Comment:
This mistake was hiding many failures! So two bugs in the tests, one hiding (mostly) the other.